components/webflow/sources/common.js

```diff
@@ -0,0 +1,77 @@
+const { v4: uuidv4 } = require("uuid");
+const webflow = require("../webflow.app");
+
+module.exports = {
+  props: {
+    db: "$.service.db",
+    http: "$.interface.http",
+    webflow,
+    siteId: {
+      type: "string",
+      label: "Site",
+      description: "The site from which to listen events",
+      async options(actions) {
```

Comment:
actions -> context ?

Reply:
Huh, you're right, I think I made a typo here. Will fix it.